### DISCUSSION TITLE
Access denied for user

### DISCUSSION THREAD

**Discussion by Xirider** [planetscale/discussion#513] [Bug Reports] [answered]
Hello!

When trying to connect locally me and another person can't get access to our db since yesterday evening, even though we didn't make any changes. Our railway.app is still able to connect to our db.

File "C:\Users\peter\anaconda3\envs\qgolem\Lib\site-packages\mysql\connector\connection.py", line 314, in _do_auth
self._auth_switch_request(username, password)
File "C:\Users\peter\anaconda3\envs\qgolem\Lib\site-packages\mysql\connector\connection.py", line 371, in _auth_switch_request
raise get_exception(packet)
sqlalchemy.exc.ProgrammingError: (mysql.connector.errors.ProgrammingError) 1045 (28000): Access denied for user 'peteralbert'
(Background on this error at: https://sqlalche.me/e/20/f405)

**Comment by jgreet** [comment, marked as answer]
For anyone else with the same problem: PlanetScale connection strings use randomly generated usernames, so seeing `Access denied for user <YOUR_LOCAL_USER>` indicates that the correct username isn't being passed to the client.
